Sources/App/Views/Error/ErrorPage+View.swift

```diff
@@ -18,7 +18,10 @@ enum ErrorPage {
                 ),
                 // Note: The copy in this header tag intentionally matches the copy in `search_core.js`.
                 .h4("Something went wrong. Sorry!"),
-                .p(.text(model.errorMessage))
+                .p(.text(model.errorMessage)),
+                .if(model.errorInstructions.count > 0,
```

Comment:
I thought this was a little cleaner than having this be optional just because we don't have easy access to if let here and so if it's optional, I also needed to add a nil coalesce on the next line down.

Reply:
There is `.unwrap` though

Reply:
i.e. that could be (with an optional),

```
.wrap(model.errorInstructions) { .p(.text($0)) }
```

Reply:
Yep, fixing it up.